dev/tasks/conda-recipes/arrow-cpp/meta.yaml

```diff
@@ -1,124 +1,237 @@
+{% set version = ARROW_VERSION %}
+{% set number = "6" %}
+{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '9.2' %}
+{% set build_ext_version = "1.0.0" %}
+{% set build_ext = "cuda" if cuda_enabled else "cpu" %}
+{% set build_string = "py{}h{}_{}_{}".format(CONDA_PY, PKG_HASH, number, build_ext) %}
+{% set proc_build_number = "1" %}
+
 package:
-  name: arrow-cpp
-  version: {{ ARROW_VERSION }}
+  name: arrow-cpp-ext
+  version: {{ version }}
 
 source:
   path: ../../../../
 
 build:
-  build:
-  number: 0
-  skip: true  # [win and vc<14]
+  number: {{ number }}
+  skip: true  # [cuda_compiler_version not in (undefined, "None", "9.2")]
   run_exports:
-    - {{ pin_subpackage("arrow-cpp", max_pin="x.x") }}
+    - {{ pin_subpackage("arrow-cpp", max_pin="x.x.x") }}
+
+outputs:
+  - name: arrow-cpp-proc
```

Comment:
Are we going to have this packages upstream as well? Is this going to work like:

```bash
conda install arrow-cpp-proc=cuda
```

I'm not that familiar with conda meta packages, is `-proc` a common naming convention?

Reply:
Yes, this package is already available upstream. `-proc`  is one of two competing naming conventions. It might be soon renamed to `arrow-cpp-mutex`.